opba-banking-protocol-facade/src/main/java/de/adorsys/opba/protocol/facade/config/EncryptionConfig.java

```diff
@@ -0,0 +1,46 @@
+package de.adorsys.opba.protocol.facade.config;
+
+import com.google.common.io.Resources;
+import com.google.crypto.tink.Aead;
+import com.google.crypto.tink.CleartextKeysetHandle;
+import com.google.crypto.tink.JsonKeysetReader;
+import com.google.crypto.tink.KeysetHandle;
+import com.google.crypto.tink.aead.AeadConfig;
+import lombok.SneakyThrows;
+import org.bouncycastle.jce.provider.BouncyCastleProvider;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.nio.file.Paths;
+import java.security.Provider;
+import java.security.SecureRandom;
+import java.security.Security;
+
+@Configuration
+public class EncryptionConfig {
+    public static final SecureRandom SECURE_RANDOM = new SecureRandom();
+    public static final int SALT_LENGTH = 8;
```

Comment:
Better to move these into ConfigurationProperties and set defaults there
```
public static final int SALT_LENGTH = 8;
...
String providerName;
```

Reply:
ok